features/having_order_partially_refunded.feature

```diff
@@ -0,0 +1,28 @@
+@refunds
+Feature: Having order partially refunded
+    In order to note that part of the order total is refunded
+    As an Administrator
+    I want to have order partially refunded
+
+    Background:
+        Given the store operates on a single green channel in "United States"
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10.00"
+        And the store has "Galaxy Post" shipping method with "$10.00" fee
+        And the store allows paying with "Space money"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought 3 "Mr. Meeseeks T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And I am logged in as an administrator
+        And the order "#00000022" is already paid
+
+    @ui
+    Scenario: Having order partially refunded when some items are refunded
+        Given 1st "Mr. Meeseeks T-Shirt" product from order "#00000022" has already been refunded with "Space money" payment
+        When I browse orders
+        Then the order "#00000022" should have order payment state "Partially refunded"
+
+    @ui
+    Scenario: Having order partially refunded when its shipping is refunded
+        Given shipment from order "#00000022" has already "$1.00" refunded with "Space money" payment
+        When I browse orders
+        Then the order "#00000022" should have order payment state "Partially refunded"
```

Comment:
It looks like this `When` step is smuggling view logic inside business scenario - browsing orders do not change the state at all. I know we use a similar structure in Sylius, but we can do better in plugins 🎉 Doesn't need to be solved in this PR though.
```gherkin
When the shipment from order XXX is refunded $1 with "Space money" payment
Then the order XXX should have...
```